packages/api/src/schema/api-schema.yaml

```diff
@@ -1957,18 +2036,11 @@ components:
           example: 1587667174725
         publicKey:
           type: string
+          example: "-----BEGIN PUBLIC KEY"
```

Comment:
This may be misleading - this should be base64 encoded

Reply:
Sure, I will remove it for now and will talk to Gioele about the correct format and example